### PR TITLE
Allow rejecting proposals from a committee.

### LINKED ISSUE #1371: Anträge zurückweisen
Anträge müssen auch zurückgewiesen werden können.

- Neue Transition
- Eintrag in History mit Kommentar
- SubmittedProposal löschen

Siehe auch #1356.

### PR DESCRIPTION
**Pull request by deiferni**
This PR adds a workflow transition to reject a proposal and revert it to pending state. It is possible to enter an optional comment indicating why the proposal was rejected.

This PR also contains an improved way to elevate a current user's privileges, it is now possible to retain the username and avoid `specialusers.system`.

Depends on https://github.com/4teamwork/plonetheme.teamraum/pull/399.

Closes #1371.

### CODE REVIEW

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po [hunk 1]
```diff
@@ -867,6 +876,11 @@ msgstr "Start"
 msgid "label_title"
 msgstr "Titel"
 
+#. Default: "Transition ${transition} executed"
+#: ./opengever/meeting/browser/meetings/transitions.py:25
+msgid "label_transition_executed"
+msgstr "Transition ${transition} ausgeführt"
```
Comment (phgross) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: Transition  --> Statusänderung ?

File: opengever/meeting/browser/proposaltransitions.py [hunk 2]
```diff
@@ -29,4 +39,56 @@ def execute_transition(self, transition_name):
...
-        return self.request.RESPONSE.redirect(self.context.absolute_url())
+        response = self.request.RESPONSE
+        if response.status != 302:  # only redirect if not already redirecting
+            return response.redirect(self.context.absolute_url())
+
+
+class IRejectProposalSchema(Interface):
+
+    text = schema.Text(
+        title=_(u'label_reject_proposal_comment', default=u'Comment'),
```
Comment (phgross) [inline opengever/meeting/browser/proposaltransitions.py]: `label_reject_proposal_comment` --> `label_reject_proposal_text`

**Comment by phgross** [comment]
@deiferni :+1: nur das eine oder andere Detail ...

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po (hunk 1 above)
Comment (deiferni) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: :+1:

File: opengever/meeting/browser/proposaltransitions.py (hunk 2 above)
Comment (deiferni) [inline opengever/meeting/browser/proposaltransitions.py]: :+1: und auch help_reject_proposal_comment -> help_reject_proposal_text :wink:

**Comment by deiferni** [comment]
@phgross aktualisiert

**Pull request merged**